components/automate-cli/cmd/chef-automate/deprecated.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// A place to put all of our deprecated commands
+
+func newAdminTokenCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:    "admin-token NAME",
+		Short:  "[deprecated] Chef Automate IAM v1 admin token create",
+		RunE:   runAdminTokenCmd,
+		Args:   cobra.ExactArgs(1),
+		Hidden: true,
+	}
+}
+
+func init() {
+	adminCommand := newAdminTokenCommand()
+	RootCmd.AddCommand(adminCommand)
+}
+
+func runAdminTokenCmd(cmd *cobra.Command, args []string) error {
+	name := args[0]
+
+	writer.Failf("The admin-token command was an IAM V1 command and is now deprecated. "+
+		"To make an admin token in IAM V2, run:\n\nchef-automate iam token create %s --admin", name)
```

Comment:
given that the original command didn't include a name, this msg needs to omit that arg. I'd recommend sticking with what we do in our docs: `chef-automate iam token create <your-token-name> --admin` or `chef-automate iam token create <token-name> --admin`